app/renderer/components/Inspector/Inspector.js

```diff
@@ -123,6 +124,9 @@ export default class Inspector extends Component {
         <Tooltip title="Search for element">
            <Button id='searchForElement' icon="search" onClick={showLocatorTestModal}/>
         </Tooltip>
+        <Tooltip title="Copy XML Source to Clipboard">
+           <Button id='btnSourceXML' icon="copy" onClick={() => clipboard.writeText(this.props.sourceXML)}/>
```

Comment:
we typically pull props out of `this.props` at the beginning of the method. then we can just refer to it as `sourceXML` here